lib/usdUfe/utils/Utils.h

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef USDUFE_UTIL_H
+#define USDUFE_UTIL_H
+
+#include <usdUfe/base/api.h>
+
+#include <string>
+
+/// General utilities for working with the UsdUfe library.
+namespace USDUFE_NS_DEF {
+
+USDUFE_PUBLIC
+std::string SanitizeName(const std::string& name);
```

Comment:
Not yours, but leading uppercase for function name is not as per coding standard.  Can do this later, obviously.

Reply:
Yes okay. I've added a note to clean this up later.